packages/components/src/components/form/Textarea/index.tsx

```diff
@@ -13,134 +15,83 @@ const Wrapper = styled.div`
     justify-content: flex-start;
 `;
 
-interface StyledTextareaProps extends BaseTextareaProps {
-    tooltipAction?: React.ReactNode;
-    width?: any;
-    state?: InputState;
-    monospace?: boolean;
-    borderWidth?: number;
-    borderRadius?: number;
-}
-
-const StyledTextarea = styled.textarea<StyledTextareaProps>`
-    width: ${props => (props.width ? `${props.width}px` : '100%')};
+const StyledTextarea = styled.textarea<Pick<TextareaProps, 'state' | 'width' | 'isMonospace'>>`
+    width: ${({ width }) => (width ? `${width}px` : '100%')};
     padding: 14px 16px;
     box-sizing: border-box;
-    border: solid ${props => props.borderWidth}px
-        ${props =>
-            props.state ? getStateColor(props.state, props.theme) : props.theme.STROKE_GREY};
-    border-radius: ${props => props.borderRadius}px;
+    border: solid 1px
+        ${({ state, theme }) => (state ? getStateColor(state, theme) : theme.STROKE_GREY)};
```

Comment:
what is state? this name is super generic and gives you no idea what it is

Reply:
Refactored in all inputs where its used